src/main/java/org/alliancegenome/curation_api/enums/ChromosomeAccessionEnum.java

```diff
@@ -0,0 +1,161 @@
+	SGD_X("chrX", "RefSeq:NC_001142.9", "R64-4-1"),
+	SGD_XI("chrXI", "RefSeq:NC_001143.9", "R64-4-1"),
+	SGD_XII("chrXII", "RefSeq:NC_001144.5", "R64-4-1"),
+	SGD_XIII("chrXIII", "RefSeq:NC_001145.3", "R64-4-1"),
+	SGD_XIV("chrXIV", "RefSeq:NC_001146.8", "R64-4-1"),
+	SGD_XV("chrXV", "RefSeq:NC_001147.6", "R64-4-1"),
+	SGD_XVI("chrXVI", "RefSeq:NC_001148.4", "R64-4-1"),
+	SGD_MT("chrmt", "RefSeq:NC_001224.1", "R64-4-1"),
+	WB_I("I", "RefSeq:NC_003279.8", "WBcel235"),
+	WB_II("II", "RefSeq:NC_003280.10", "WBcel235"),
+	WB_III("III", "RefSeq:NC_003281.10", "WBcel235"),
+	WB_IV("IV", "RefSeq:NC_003282.8", "WBcel235"),
+	WB_V("V", "RefSeq:NC_003283.11", "WBcel235"),
+	WB_X("X", "RefSeq:NC_003284.9", "WBcel235"),
+	WB_MT("MtDNA", "RefSeq:NC_001328.1", "WBcel235"),
+	ZFIN_1("1", "RefSeq:NC_007112.7", "GRCz11"),
+	ZFIN_2("2", "RefSeq:NC_007113.7", "GRCz11"),
+	ZFIN_3("3", "RefSeq:NC_007114.7", "GRCz11"),
+	ZFIN_4("4", "RefSeq:NC_007115.7", "GRCz11"),
+	ZFIN_5("5", "RefSeq:NC_007116.7", "GRCz11"),
+	ZFIN_6("6", "RefSeq:NC_007117.7", "GRCz11"),
+	ZFIN_7("7", "RefSeq:NC_007118.7", "GRCz11"),
+	ZFIN_8("8", "RefSeq:NC_007119.7", "GRCz11"),
+	ZFIN_9("9", "RefSeq:NC_007120.7", "GRCz11"),
+	ZFIN_10("10", "RefSeq:NC_007121.7", "GRCz11"),
+	ZFIN_11("11", "RefSeq:NC_007122.7", "GRCz11"),
+	ZFIN_12("12", "RefSeq:NC_007123.7", "GRCz11"),
+	ZFIN_13("13", "RefSeq:NC_007124.7", "GRCz11"),
+	ZFIN_14("14", "RefSeq:NC_007125.7", "GRCz11"),
+	ZFIN_15("15", "RefSeq:NC_007126.7", "GRCz11"),
+	ZFIN_16("16", "RefSeq:NC_007127.7", "GRCz11"),
+	ZFIN_17("17", "RefSeq:NC_007128.7", "GRCz11"),
+	ZFIN_18("18", "RefSeq:NC_007129.7", "GRCz11"),
+	ZFIN_19("19", "RefSeq:NC_007130.7", "GRCz11"),
+	ZFIN_20("20", "RefSeq:NC_007131.7", "GRCz11"),
+	ZFIN_21("21", "RefSeq:NC_007132.7", "GRCz11"),
+	ZFIN_22("22", "RefSeq:NC_007133.7", "GRCz11"),
+	ZFIN_23("23", "RefSeq:NC_007134.7", "GRCz11"),
+	ZFIN_24("24", "RefSeq:NC_007135.7", "GRCz11"),
+	ZFIN_25("25", "RefSeq:NC_007136.7", "GRCz11"),
+	ZFIN_MT("MT", "RefSeq:NC_002333.2", "GRCz11");
+
+
+	public String chromosomeName;
+	public String accession;
+	public String assemblyIdentifier;
+
+	private ChromosomeAccessionEnum(String chromosomeName, String accession, String assemblyName) {
+		this.chromosomeName = chromosomeName;
+		this.accession = accession;
+		this.assemblyIdentifier = assemblyName;
+	}
+
+	public static ChromosomeAccessionEnum getChromosomeAccessionEnum(String chromosomeAccession) {
+		if (chromosomeAccession == null) {
+			return null;
+		}
+		for (ChromosomeAccessionEnum chromosome : values()) {
+			if (chromosomeAccession.equals(chromosome.accession)) {
+				return chromosome;
+			}
+		}
+		return null;
+	}
+	
+	public static String getChromosomeAccession(String chromosomeName, String assemblyIdentifier) {
+		if (chromosomeName == null || assemblyIdentifier == null) {
+			return null;
+		}
+		for (ChromosomeAccessionEnum chromosome : values()) {
+			if (chromosomeName.equals(chromosome.chromosomeName) &&
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.OperatorWrapCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'&&' should be on a new line.
<!-- __reviewdog__:ChBkZDk1MWYzZGE4NjJhNTBiEgpjaGVja3N0eWxl -->